rom/dev/src/flow/update_reset.rs

```diff
@@ -211,7 +211,12 @@ impl UpdateResetFlow {
     fn populate_data_vault(data_vault: &mut DataVault, info: &ImageVerificationInfo) {
         data_vault.write_warm_reset_entry48(WarmResetEntry48::RtTci, &info.runtime.digest.into());
 
-        data_vault.write_warm_reset_entry4(WarmResetEntry4::RtSvn, info.runtime.svn);
+        // TODO: calculate min-SVN
```

Comment:
Same question, why todo

Reply:
Was trying to avoid more functional changes in this PR.